dist/index.js

```diff
@@ -62,7 +62,7 @@ try {
     const home = process.env['HOME'];
     const homeSsh = home + '/.ssh';
 
-    const privateKey = core.getInput('ssh-private-key').trim();
+    const privateKey = core.getInput('ssh-private-key');
```

Comment:
Could you rebase this PR against `master` please? Then this change should go away.